stack/lambda_bundling.py

```diff
@@ -0,0 +1,39 @@
+import os
+import zipfile
+from subprocess import check_call
+from sys import executable
+
+
+def lambda_pip_install_requirements(lambda_directory: str, requirements_file: str) -> None:
+    # Documentation recommend against calling pip internal api; rather, via command line
+    # https://pip.pypa.io/en/latest/user_guide/#using-pip-from-your-program
+
+    check_call(
+        [
+            executable,
+            "-m",
+            "pip",
+            "install",
+            "--quiet",
+            "--force-reinstall",
+            f"--target={lambda_directory}/packages",
+            f"--requirement={requirements_file}",
+        ]
+    )
+
+
+def zip_lambda_assets(lambda_directory: str) -> str:
+    lambda_assets = f"{lambda_directory}/lambda.zip"
+
+    with zipfile.ZipFile(lambda_assets, "w") as zipped_lambda_assets:
+        for path, _directories, files in os.walk(lambda_directory):
+            if f"{lambda_directory}/packages" in path:
```

Comment:
Would it be possible to avoid this logic, by putting the packages in a standard location? There's some good [documentation](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-lambda-python-alpha-readme.html#packaging) which might help.

Reply:
Yep. It is a bit messy. I have changed how this is handled. 
I am not in favour of using 'aws-lambda-python-alpha' module since it is still in experimental (alpha). This has already caused a problem few months back when they update how files are copied in lambda, breaking geostore deployment.